Sources/XcodeProj/Objects/Project/PBXProj.swift

```diff
@@ -284,13 +284,24 @@ extension PBXProj: Equatable {
 // MARK: - Writable
 
 extension PBXProj: Writable {
+    public func dataRepresentation(outputSettings: PBXOutputSettings) throws -> Data? {
+        let encoder = PBXProjEncoder(outputSettings: PBXOutputSettings())
```

Comment:
```suggestion
        let encoder = PBXProjEncoder(outputSettings: outputSettings)
```

Reply:
@kwridan Thanks for pointing this out, I have fixed this now.